addon/components/frost-password.js

```diff
@@ -67,17 +67,12 @@ export default Component.extend(FrostEventsProxy, {
       isRevealed: false,
       revealable: false,
       tabindex: 0,
-      value: null,
```

Comment:
The initialization of these properties to null was causing a warning message to be shown in the console when running the demo app. An issue has been created: https://github.com/ciena-frost/ember-frost-core/issues/364 to address adding these properties back in and configuring prop-types so that the message is not shown in the console.